api/app/Policies/ClassificationPolicy.php

```diff
@@ -53,7 +53,7 @@ public function create(User $user)
      */
     public function update(User $user, Classification $classification)
     {
-        return $user->isAdmin();
+        return $user && $user->isAbleTo("update-any-classification");
```

Comment:
can we remove these unused params? classification , department , skillFamily etc ?

Reply:
Removed in [add note, remove unused params](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5801/commits/4af9141d0e361ce1acecfca2bf73f760613962cf)